impl/src/main/java/io/quarkiverse/kafkastreamsprocessor/impl/TopologyProducer.java

```diff
@@ -136,8 +136,8 @@ public TopologyConfigurationImpl configuration(BeanManager beanManager,
         TopologyConfigurationImpl configuration = initializeConfiguration(beanManager);
         // Step 2: apply default configuration
         defaultConfiguration.apply(configuration);
-        // Step 3: if an alternative customizer is provided, then apply it (default does nothing)
-        configCustomizer.fillConfiguration(configuration);
+        // Step 3: if alternative customizers are provided, then apply them (default does nothing)
+        configCustomizers.forEach(customizer -> customizer.fillConfiguration(configuration));
```

Comment:
Why not applying the the customizer with the most priority ? 
Applying each customizer might end in a mix of everything